lib/msf/core/post/linux/packages.rb

```diff
@@ -0,0 +1,74 @@
+# -*- coding: binary -*-
+
+module Msf
+  class Post
+    module Linux
+      module Packages
+        include ::Msf::Post::Linux::System
+
+        #
+        # Determines the version of an installed package
+        #
+        # @param package The package name to check for
+        # @return [Rex::Version] nil if OS is not supported or package is not installed
+        #
+        def installed_package_version(package)
+          info = get_sysinfo
+
+          if ['debian', 'ubuntu'].include?info[:distro]
+            package_version = cmd_exec("dpkg -l #{package} | grep \'^ii\'")
```

Comment:
```suggestion
            package_version = cmd_exec("dpkg-query -f='${Version}' -W #{package}")
```

This will directly output the version of the package if it's installed